Projects/App/Sources/Present/Detail/ViewController/Modal/VC/DetailOptionModalViewController.swift

```diff
@@ -0,0 +1,62 @@
+import UIKit
+import SnapKit
+import Then
+import RxSwift
+import RxCocoa
+
+final class DetailOptionModalViewController: UIViewController {
+    private let disposeBag = DisposeBag()
+    
+    let optionList = [
+        OptionData(image: UIImage(systemName: "exclamationmark.circle")!,
+                   text: "게시물 신고",
+                   color: UIColor.systemRed),
+        OptionData(image: UIImage(systemName: "person.crop.circle.badge.xmark")!,
+                   text: "사용자 차단",
+                   color: UIColor.black),
+        OptionData(image: ChoiceAsset.Images.instarIcon.image,
+                   text: "Instagram에 공유",
+                   color: UIColor.black)
+    ]
+    
+    let optionData = BehaviorRelay<[OptionData]>(value: [])
+    
+    private let optionListTableView = UITableView().then {
+        $0.selectionFollowsFocus = false
+        $0.isScrollEnabled = false
+        $0.rowHeight = 65
+        $0.register(DetailOptionTableViewCell.self,
+                    forCellReuseIdentifier: DetailOptionTableViewCell.identifier)
+    }
+    
+    override func viewDidLoad() {
+        view.backgroundColor = .white
+        
+        optionData.accept(optionList)
+        
+        bindTableView()
+        addView()
+        setLayout()
+    }
+    
+    private func bindTableView() {
+        optionData
+            .bind(to: optionListTableView.rx.items(
+                cellIdentifier: DetailOptionTableViewCell.identifier,
+                cellType: DetailOptionTableViewCell.self)) { (row, data, cell) in
+                    cell.configure(data: data)
+                    cell.selectionStyle = .none
+                }.disposed(by: disposeBag)
+    }
+    
+    private func addView() {
+        view.addSubview(optionListTableView)
+    }
+    
+    private func setLayout() {
+        optionListTableView.snp.makeConstraints {
+            $0.top.equalTo(view.safeAreaLayoutGuide).offset(10)
+            $0.width.height.equalToSuperview()
+        }
```

Comment:
top 을 safeAreaGuide에서 10만큼 offset을 줬는데,
width, height 를 다시 superView 로 제약을 주면 의미가 있는건가요?

Reply:
서로 독립적이기 때문에 잘 동작은 합니다만, 좋지 않은 레이아웃 설정인것 같습니다.
```suggestion
        optionListTableView.snp.makeConstraints {
            $0.top.equalTo(view.safeAreaLayoutGuide).offset(10)
            $0.bottom.leading.trailing.equalToSuperView()
        }
```
어떨까요 😏

Reply:
아하 그렇군요
좋은 제안인 것 같습니다!! 👍🏻